templates/download/raspberry-pi/index.html

```diff
@@ -140,102 +140,84 @@ <h4 class="u-no-margin--bottom">Ubuntu {{ releases.lts.full_version }} LTS</h4>
 <section class="p-strip is-bordered is-shallow u-no-padding--bottom">
   <div class="row u-equal-height">
     <div class="col-3">
-      <h4>Ubuntu {{ releases.latest.full_version }}</h4>
-      <p>Get the latest version of Ubuntu with nine months support, until {{ releases.latest.eol }}.</p>
+      <h4>Ubuntu Desktop {{ releases.latest.full_version }}</h4>
```

Comment:
Should this content by in a conditional checking if the latest is the latest or are we on an LTS release?

Reply:
that's on line 139